css-cascade-6/Overview.bs

```diff
@@ -476,6 +476,9 @@ Scoping Styles: the ''@scope'' rule</h4>
 	are [=scoped selectors|scoped by=]
 	the selectors of the outer one.
 
+	[=Pseudo-elements=] cannot be [=scoping roots=] or [=scoping limits=];
```

Comment:
Should this move up into `2.5 Scoped Styles` where `<scope-start>` and `<scope-end>` are defined? Merging as-is, but might want to come back and move this, so making note of it…